Modules/Filtering/ImageFeature/include/itkHessianToObjectnessMeasureImageFilter.h

```diff
@@ -140,13 +140,13 @@ class ITK_TEMPLATE_EXPORT HessianToObjectnessMeasureImageFilter:public
 
 private:
   // functor used to sort the eigenvalues are to be sorted
-  // |e1|<=|e2|<=...<=|eN|
```

Comment:
The original statement is correct. As there can be Eigen values were are the same.

Reply:
You are right. Amending...